back-end/src/sciler/config/workingConfigTypes.go

```diff
@@ -261,6 +268,19 @@ func compare(param1 interface{}, param2 interface{}, comparision string) bool {
 	}
 }
 
+// equalWithTypo checks if two strings are roughly equal
+// it tries to ignore typos
+func equalWithTypo(expected string, actual string) bool {
+	return checkStringDistanceConstraint(expected, actual, int(math.Max(1, float64(len(expected))/5)))
+}
```

Comment:
please document the maxDifference, how much in practice it can differ and why that is chosen

Reply:
done